workflows/rules/common.smk

```diff
@@ -0,0 +1,44 @@
+"""
+rule module: common.smk
+
+common.smk is a workflow module that setups the expected input and output paths
+for the main analytical workflow.
+"""
+from snakemake.io import expand
+from cytosnake.helpers import helper_funcs as hf
+from cytosnake.utils.config_utils import load_data_path_configs, load_meta_path_configs
+
+# Paths
+DATA_DIR = str(load_data_path_configs())
+
+# ------
+# INPUTS
+# ------
+# -- generating a wild card list (just the file base names)
+plate_name = hf.get_file_basenames(DATA_DIR, ext_target="sqlite")
```

Comment:
will this always be `sqlite`? Probably ok for now, just  food for thought!

Reply:
For now yes. Once we create a structured config file, we can start removing the hard coded values into the config file!